pkg/pillar/types/global.go

```diff
@@ -27,6 +27,13 @@ const (
 	SenderStatusCertUnknownAuthorityProxy              // device configed proxy, may miss proxy certificate for MiTM
 )
 
+const (
+	MinuteInSec = 60
+	HourInSec   = 60 * MinuteInSec
+	DayInSec    = 24 * HourInSec
+	YearInSec   = 365 * DayInSec
+)
+
```

Comment:
Don't these already exist in the time package?

Reply:
The types are different. We only support a generic u32 as values.. The ones in time package are of type time.Duration